### PR TITLE
sdk how to document - add v1 serialization code examples

### PR DESCRIPTION
**Pull request by harleenkohli**

### CODE REVIEW

File: docs/how-to.md [hunk 1]
```diff
@@ -19,14 +21,43 @@ SarifLog log = ... ;
...
 ```
+```C#
+# For SARIF V1 File:
+var settings = new JsonSerializerSettings()
+{
+    ContractResolver = SarifContractResolverVersionOne.Instance,
+    Formatting = Formatting.Indented
+};
+
+SarifLog log = ... ;
```
Comment (lgolding) [inline docs/how-to.md]: >SarifLog [](start = 0, length = 8)

`SarifLogVersionOne`

File: docs/how-to.md [hunk 2]
```diff
@@ -9,7 +9,9 @@
 ## Write a SARIF log file to disk
 
 ```C#
-JsonSerializerSettings settings = new JsonSerializerSettings()
+
+# For SARIF V2 File:
```
Comment (lgolding) [inline docs/how-to.md]: >For [](start = 2, length = 3)

"For a file in the standardized SARIF v2.1.0 format:"

File: docs/how-to.md [hunk 3]
```diff
@@ -19,14 +21,43 @@ SarifLog log = ... ;
 string sarifText = JsonConvert.SerializeObject(log, settings);
 File.WriteAllText(outputFilePath, sarifText);
 ```
+```C#
+# For SARIF V1 File:
```
Comment (lgolding) [inline docs/how-to.md]: >For [](start = 2, length = 3)

"For a file in the deprecated, pre-standardization SARIF v1.0 format:"

**Comment by lgolding** [comment]
> ```C#

It looks like you accidentally duplicated these four lines.

---
Refers to: docs/how-to.md:64 in 56ab696. [](commit_id = 56ab696dbf8d551516fe66a47892129b8d45ce33, deletion_comment = False)

File: docs/how-to.md [hunk 4]
```diff
@@ -19,14 +21,43 @@ SarifLog log = ... ;
...
+string logContents = File.ReadAllText(logFilePath);
+
+var settings = new JsonSerializerSettings()
+{
+    ContractResolver = SarifContractResolverVersionOne.Instance
+};
+
+SarifLog log = JsonConvert.DeserializeObject<SarifLog>(logContents, settings);
```
Comment (lgolding) [inline docs/how-to.md]: >SarifLog [](start = 0, length = 8)

SarifLogVersionOne``

Comment (harleenkohli) [reply docs/how-to.md]: oh my! yes this is indeed incorrect! 

---
In reply to: [301325750](https://github.com/microsoft/sarif-sdk/pull/1566#discussion_r301325750) [](ancestors = 301325750)

**Pull request merged**